### PR TITLE
[full-ci] chore: prepare changelog, bump version, update CD

### PR DESCRIPTION
**Pull request by micbar**
## Description

- [x] Run Full CI
- [x] Create Final Tag
- [x] Merge back into `master`

## Related Issue

- Fixes <issue_link>

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by micbar** [comment]
> you forgot to add the changelog folder ;-)

Ok, added it.

**Review by kulmann — APPROVED** [review]

**Review by ScharfViktor — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,8 +17,8 @@
 * [Changelog for 3.0.0](#changelog-for-300-2023-06-06)
 * [Changelog for 2.0.0](#changelog-for-200-2022-11-30)
 * [Changelog for 1.20.0](#changelog-for-1200-2022-04-13)
-* [Changelog for 1.19.1](#changelog-for-1191-2022-03-29)
 * [Changelog for 1.19.0](#changelog-for-1190-2022-03-29)
+* [Changelog for 1.19.1](#changelog-for-1191-2022-03-29)
```
Comment (phil-davis) [inline CHANGELOG.md:21]: That's a strange change, the list goes from latest to earliest.

File: CHANGELOG.md [hunk 2]
```diff
@@ -8900,6 +8890,24 @@ The following sections list the changes for 1.19.0.
    https://github.com/owncloud/ocis/pull/3375
    https://github.com/owncloud/web/releases/tag/v5.3.0
 
+# Changelog for [1.19.1] (2022-03-29)
+
+The following sections list the changes for 1.19.1.
+
+[1.19.1]: https://github.com/owncloud/ocis/compare/v1.18.0...v1.19.1
```
Comment (phil-davis) [inline CHANGELOG.md:8897]: This goes from 1.18.0 to 1.19.1 (jumping 1.19.0)

File: CHANGELOG.md [hunk 3]
```diff
@@ -8704,29 +8712,11 @@ The following sections list the changes for 1.20.0.
...
-   URLs for Special items (space image, readme) were broken.
-
-   https://github.com/owncloud/ocis/pull/3419
-
 # Changelog for [1.19.0] (2022-03-29)
 
 The following sections list the changes for 1.19.0.
 
-[1.19.0]: https://github.com/owncloud/ocis/compare/v1.18.0...v1.19.0
+[1.19.0]: https://github.com/owncloud/ocis/compare/v1.19.1...v1.19.0
```
Comment (phil-davis) [inline CHANGELOG.md:8719]: This is a "backwards" compare. Github says:
There isn’t anything to compare.
v1.19.1 is up to date with all commits from v1.19.0. Try [switching the base](https://github.com/owncloud/ocis/compare/v1.19.0...v1.19.1) for your comparison.

**Comment by micbar** [comment]
All changelog.md changes will be overwritten by CI after the merge. Have been commited accidentally.

**Pull request merged**